### DISCUSSION TITLE
Hello, now we know that openmatadata supports the function of metadata collection. Is there any plan to add the function of metadata registration and data directory in the future?

### DISCUSSION THREAD

**Discussion by fanyo190** [open-metadata/OpenMetadata#10972] [Q&A]
Hello, now we know that openmatadata supports the function of metadata collection. Is there any plan to add the function of metadata registration and data directory in the future?

**Comment by pmbrull** [comment]
Hi @fanyo190, thanks for reaching out. Could you please expand a bit more on what do you mean about metadata registration and data directory?

With the current governance and discovery features, folks in an organization should be able to navigate through any data asset and use OM as a centralized repository.

Thanks
